integration-test/src/main/java/com/sequenceiq/it/cloudbreak/ClusterTestFactory.java

```diff
@@ -12,11 +12,12 @@
 
 public class ClusterTestFactory extends CloudbreakTest {
     @Factory
-    @Parameters({"provider"})
-    public Object[] clusterTestFactory(@Optional(GcpCloudProvider.GCP) String provider) {
-        CloudProvider[] providers = CloudProviderHelper.providerFactory(provider);
-        Object[] results = new Object[providers.length];
-        IntStream.range(0, providers.length).forEach(index -> results[index] = new ClusterTests(providers[index]));
+    @Parameters({"providers"})
+    public Object[] clusterTestFactory(@Optional(OpenstackCloudProvider.OPENSTACK) String providers) {
+        CloudProvider[] cloudProviders = CloudProviderHelper.providerFactory(providers, getTestParameter());
+        Object[] results = new Object[cloudProviders.length];
+        IntStream.range(0, results.length).forEach(index -> results[index] = new ClusterTests(cloudProviders[index],
```

Comment:
I would use "Arrays.stream(providers).map(ClusterTests::new).toArray()" this expression for initialization.

Reply:
ok